ci_framework/roles/hive/README.md

```diff
@@ -0,0 +1,123 @@
+## Role: hive
```

Comment:
```suggestion
# hive
```
The README.md layout has changed a bit in order to match sphinx requirements.

Please also remove one level of `#` in the other titles, so that it will display nicely.